bin/webpack.js

```diff
@@ -1,19 +1,59 @@
 #!/usr/bin/env node
 
+const {
+	exec
+} = require("child_process");
+const inquirer = require("inquirer");
+
+function runCommand(command) {
```

Comment:
can this be made into arrow function?

Reply:
Well, it can be made, since there is no this, new, or scope related things.

Reply:
Well, yeah, it can be made.

Reply:
Well, yes, there is no scope related things here. Is there any reason that we could prefer to use arrow functions here?